Fakebook.Posts/Fakebook.Posts.DataAccess/Fakebook.Posts.DataAccess.csproj

```diff
@@ -9,6 +9,17 @@
     <Folder Include="Repositories\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="5.0.1" />
```

Comment:
pretty sure `Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore` and `Microsoft.VisualStudio.Web.CodeGeneration.Design` don't belong here, belong in the api project instead if anywhere

Reply:
@harelfishbein could you explain what these packages were for? I just included them in because they were in your previous branch.

Reply:
These packages have been removed from the DataAccess project.